scripts/qa_crowbarsetup.sh

```diff
@@ -2594,6 +2594,9 @@ function custom_configuration
             if [[ $hacloud = 1 ]] ; then
                 echo "Warning: Monasca currently is not HA ready"
             fi
+            if iscloudver 9plus && [[ $monasca_tsdb = cassandra ]]; then
```

Comment:
the variable name does not match the usual pattern of want_XXX which is automatically exported from mkcloud to qa_crowbarsetup.sh. you will have to pass it through manually in https://github.com/SUSE-Cloud/automation/blob/master/scripts/mkcloud#L294 and other places

Reply:
Thanks, fixed in [scripts/mkcloud](https://github.com/SUSE-Cloud/automation/pull/3264/files#diff-4da4ef75e4826a188f831da36089295cR341). Could you point to *"other places"*? I'm not sure what I should look after.

Reply:
I am not 100% sure, so far I was always following the want_xxx pattern. I would check which places are using variables like networkoptions that are also not matching the pattern. I think there was a place where it is reporting the effective values in case of failure for example

Reply:
I haven't found anything relevant for Monasca barclamp.

Reply:
@jdsn @bmwiedemann is that ok to introduce a new variable in mkcloud and pass it to qa_crowbarsetup.sh ? Or should we use the `want_` mechanism to do that?

Reply:
@toabctl yes, thats a good candidate for a `$want_` variable

Reply:
Done